tools/development/helpers/test-python.sh

```
@@ -7,6 +7,7 @@ test_directory="${project_directory}/bindings/python/test"
 
 pushd "${test_directory}" > /dev/null
 
-python${OSTK_PYTHON_VERSION} -m pytest -sv ${@}
+    python${OSTK_PYTHON_VERSION} -m pytest -sv ${@} \
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)